articles/errors/deprecation-errors.md

```diff
@@ -89,8 +89,6 @@ Click on the **TRY** button. If successful, you should see a screen similar to t
 | --- | --- |
 | Either calling the /ssodata directly or using old versions of embedded Lock or Auth0.js SDK to call a function which called the /ssodata endpoint. | [Migrate to Universal Login](/guides/login/migration-embedded-universal) or [migrate to Lock v11 or Auth0.js v9](/migrations#introducing-lock-v11-and-auth0-js-v9). |
 
-## Legacy Lock API notice
+## Legacy Lock API troubleshooting
 
-If you see the warning panel `Your tenant has the Legacy Lock API enabled. Please follow our Deprecation Guide then disable the Legacy Lock API in your advanced settings. The Legacy Lock API will be removed on July 16th, 2018 and your applications will no longer work if you have not fully migrated.` when you login to the Dashboard, it is because you have not turned off the Legacy Lock API switch in your [advanced settings](${manage_url}/#/tenant/advanced). If you are unaffected by the migration, or have completed it, you should be able to turn that setting off and remove the warning.
-
-If you are unsure if you are safely migrated yet, you can follow the [Deprecation Guide](/migrations/guides/legacy-lock-api-deprecation) to be sure.
+For tenant log entries regarding the Legacy Lock API, the referrer as well as information about the SDK being used are returned in the entry, in order to assist with troubleshooting. This information can help narrow down which applications might still be using outdated libraries, and which libraries they are using.
```

Comment:
Tenant log entries regarding the Legacy Lock API include the referrer and the SDK used. With this information you can see if any of your applications use outdated libraries.